packages/wallet-sdk/src/core/type/index.ts

```diff
@@ -1,5 +1,17 @@
 // Copyright (c) 2018-2024 Coinbase, Inc. <https://www.coinbase.com/>
 
+import { ProviderInterface } from './ProviderInterface';
+
+interface LegacyProviderInterface extends ProviderInterface {
+  setAppInfo?(appName: string, appLogoUrl: string | null): void;
+}
```

Comment:
let's just remove this. `setAppInfo` doesn't need to be exposed. it's being used internally only at one location. we can check it's availability directly there